Synks p magazine masterpage

stevesynk opened the pull request:
# Pull Request


## Synks p magazine masterpage

**Completed Work**

- Created new procedure: 'p-magazine-masterpage.md'
- Procedure's Goal: To create a "master page" for their magazine layout. The master page essentially acts as a template, so that users can insert elements like text and images in a consistent manner throughout their document.

## User Scenario

New user is using Adobe InDesign to create their first issue of a magazine, meant to feature their city's local music scene.

## Notable Review Requests

- There's a lot of commits bundled into this, including folder management, but the important thing is to look at the file 'p-magazine-masterpage.md'
- I created headers, as well as adding lists of instructions for the procedure, and staging/introductory paragraphs.
- Please note the level of detail, and if you think it is appropriate for a begging user.
- I still need to add images

MBethmorris commented:
Maybe the types of kits become links. Find one to focus on and then link as options to the others. 
I really like the Caution info. That is helpful.
Limited mobility isn't something I was thinking about while reading this so that is an interesting question. That might not be the audience.

Alternative methods work. It helps add credibility. 
I do think more photos would be helpful Especially with application process and scraping.

civplayer commented:
Adding more pictures is helpful, I think listing the tools is important but perhaps that section could be trimmed so only the info which is absolutely needed is included

stevesynk committed:
renamed procedure folders

stevesynk committed:
folder maintenance

stevesynk committed:
Create PR template

stevesynk committed:
added headers

stevesynk committed:
Headlines, paragraphs, and lists

Alli-Wray commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+
+    Once you have assembled the content for your magazine, like images and articles, we recommend creating handrawn sketches to determine the layout. While things may change as you work on your document, having a sketched version of the magazine will help you determine how many pages to add to the document, and how may columns of text you will need on each page. 
+
+## Prequisites
```
I'm curious as to what level of "beginner" you are aiming for here. Creating the document is on par with beginner friendly prerequisites, but the page dimensions, bleed and margins could be included in your main steps if you want to focus on new users.

Alli-Wray commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+2. In the upper section of the panel, click the _Create New Master Page_ icon at the top right of the panel. 
+    [add image]
+3. Add a name for the master page in the dialog box that appears.
+4. Click "OK"
```
Adding a success condition at the end of each major heading/step could be helpful in orienting the user before continuing.

stevesynk replied:
Great point, I think that would help a lot with clarity.

Alli-Wray commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+## Apply the master page
+1. Go back to the _Pages_ panel, and select the master page at the top of the panel. It will be indicated by a letter, so your first master page will be "A".
+2. Click on the master page thumbnail and drag it down to the thumbnail for the desired document page.
+3. Apply the master page by releasing the mouse button when a black rectangle appears on the border of the page.
```
See above comment regarding success conditions.

Alli-Wray commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+1. In the _Pages_ panel, double-click the newly created master page to open it in your window.
+2. Go to the menu and select _Layout_ > _Margins and Columns_
+3. In the dialog box, type the desired number of columns and set the spacing between the columns, known as "gutters".
+4. Check the box marked "Preview" so that you can view your guides on the page as your create them.
```
See success condition comment above.

Alli-Wray commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+1. Go back to the _Pages_ panel, and select the master page at the top of the panel. It will be indicated by a letter, so your first master page will be "A".
+2. Click on the master page thumbnail and drag it down to the thumbnail for the desired document page.
+3. Apply the master page by releasing the mouse button when a black rectangle appears on the border of the page.
+
```
I think the staging and coaching are in good shape. Are there any warnings/alerts that could be included? Any weird/finicky settings that the new user should be made aware of?

stevesynk replied:
TBH I don't know because I've never really done this haha. I am about as experienced as the user it is directed to. But hopefully I will have time before submitting to open up InDesign and test my procedure and see if there are any obvious pitfalls.

Alli-Wray reviewed (commented):
Overall, I think the set up is in good shape right now. The staging and coaching are set up well, but I did add a few comments to possibly include success conditions and add some alerting notes.

jknight6 commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+
+### Why do I need a master page?
+
+    Master pages in Indesign help you maintain consistency in your magazine layout by providing guidelines for the placement of images and text. 
```
I think the front section might flow better if instead of two subheadings structured around FAQs, there was an intro paragraph of 2-3 sentences that set the stage for master pages and how they are useful.

jknight6 commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+
+### How do I get started?
+
+    Once you have assembled the content for your magazine, like images and articles, we recommend creating handrawn sketches to determine the layout. While things may change as you work on your document, having a sketched version of the magazine will help you determine how many pages to add to the document, and how may columns of text you will need on each page. 
```
This may work better as an alerting move somewhere in the procedure - maybe in the prerequisites?

jknight6 commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+
+### How do I get started?
+
+    Once you have assembled the content for your magazine, like images and articles, we recommend creating handrawn sketches to determine the layout. While things may change as you work on your document, having a sketched version of the magazine will help you determine how many pages to add to the document, and how may columns of text you will need on each page. 
```
"handrawn" should be hyphenated: "hand-drawn"

stevesynk replied:
good catch, did not know that.

jknight6 commented on synks/p-magazine-masterpage.md:
```diff
@@ -0,0 +1,38 @@
+- Configure setting for your desired page dimensions and number of pages.
+- Add bleed and margins, so you know where images and text frames should begin and end.
+
+## Create a new master page
```
For the steps, choose whether you want to end each step with a period or not, then use that consistently throughout. I am partial to no periods at the end of steps, but as long as it's consistent it's up to you.

stevesynk replied:
Great comment. I usually go with periods, but it seems like they are maybe not the standard in this style of writing.

stevesynk committed:
Intro and Edits

-deleted FAQ style introduction headers, and added "Introduction" header and first paragraph
-changed "master page" to "parent page"
-edited steps for clarity

stevesynk committed:
added user scenario

stevesynk committed:
added reflection

lingeringcode committed:
Feedback from Lindgren

lingeringcode committed:
Lindgren: question about missing troubleshooting

stevesynk committed:
applied edits

stevesynk committed:
applied edits

stevesynk committed:
applied edits

stevesynk committed:
Update p-magazine-masterpage.md

stevesynk committed:
Update p-magazine-masterpage.md

stevesynk committed:
Update p-magazine-masterpage.md

stevesynk committed:
Update p-magazine-masterpage.md

stevesynk committed:
created dita version

stevesynk committed:
renamed file

stevesynk committed:
Update p-magazine-images.md
